Update instructions (#23)
pennylane-hq/jean_test_front: README.md

# jean_test_front

> This repository contains the guidelines for the frontend interview question, as well as a repository skeleton with which to start. It provides a bootstrapped MVP to help you get started quickly.

Reviewer: ```suggestion
This repository contains the guidelines for the frontend interview question as well as a bootstrapped MVP to help you begin quickly.
```

 
We do not expect the prototype to be UI-rich as we'll mainly focus on code quality & user experience. We expect you to adopt standard coding practices & setup, including testing, as if you were working on a real application with other coworkers.

The project comes with several pre-installed libraries to help you get started. We encourage you to leverage these existing dependencies rather than adding new ones, unless you have a legitimate need that cannot be met with the current setup.

Notable pre-installed libraries include:

- React Hook Form - For handling form state and validation
- Yup - For schema validation

> If you're not familiar with React Hook Form or Yup, feel free to use alternative form handling approaches you're more comfortable with. For other use cases, if adding any new dependencies, please be prepared to justify why they were necessary for your implementation.

Reviewer: I think this might be confusing / some things are said twice when I re-read the paragraph above:

```md
The project comes with several pre-installed libraries to help you get started. We encourage you to leverage these existing dependencies rather than adding new ones, unless you have a legitimate need that cannot be met with the current setup.
```

Shouldn't we try to merge both?

Reviewer: Way better IMO, thank you very much!

- A potential prototype implementation (feel free to work around API limitations)
- What might be missing for you to implement it (API limitations, technical constraints)

You are welcome and encouraged to use AI tools to assist you in completing this case study. To maintain transparency:

- Please document which AI tools you utilized during the assignment
- For each AI tool used, briefly describe:
  - The main tasks or problems where you applied it
  - How you validated and refined the AI-generated code

> Note: While AI can be a valuable assistant, ensure you fully understand the implemented code and can explain your technical decisions.

Reviewer 2: ❤️

### API client

> An API client based on `openapi-client-axios` is available through a React Context set up in `src/app/index.tsx`. The context can be consumed using the `useApi` hook. Before using it, please add the token you received in `.env.local`. If you do not have one, please contact us.

Author: I made a mistake by committing a token. To avoid such issue later (and for our candidates) I changed the instructions.

ReactDOM.render(
  <ApiProvider
    url="https://jean-test-api.herokuapp.com/"

>     token="eb8ff9bc-e737-4505-aa57-f01507642073" // set your api token here

Reviewer 2: Which token is this?

Author: I should remove it indeed. sorry for that
